docs/CEAP_en_US.md

```diff
@@ -0,0 +1,100 @@
+Article 1 - It can use, above the calculated quota for it's State, an extra of:
```

Comment:
Lot's of change needed here:

1. In the English language it's not common to invert the order of the sentence with the help of commas, so I wouldn't interpolate
2. In that case it is not **it's** (contraction of _it is_), it it **its** (possessive)
3. We do not refer to a given congressperson as **it**, but maybe as **one** or the impessoal **they**
4. State with capital letter usually refers to the concept of a (national) state, states as units of a federation does not need capital letter

That said I would rephrase it as _Above the calculated quota for their state they can use:_.